service/src/test/java/bio/terra/externalcreds/services/PassportServiceTest.java

```diff
@@ -109,12 +109,14 @@ void testGetGA4GHPassportLinkedAccountNoPassport() {
 
     @Test
     void testValidPassportMatchingCriteria() throws URISyntaxException {
+      when(externalCredsConfigMock.getAllowedJwtAlgorithms()).thenReturn(List.of("RS256", "ES256"));
```

Comment:
Just one quick suggestion: I think that instead of mocking this in each test, you could just mock it once in the helper function they all call ([here](https://github.com/DataBiosphere/terra-external-credentials-manager/blob/8a4ece69617b94505e8b6c64f1e144ddd306c22b/service/src/test/java/bio/terra/externalcreds/services/PassportServiceTest.java#L207)).